### ISSUE TITLE
Error: 13 INTERNAL: Failed to parse server response when querying action receipt

### ISSUE THREAD

**Issue by simonerom** [iotexproject/iotex-antenna#158]
Using current iotex-antenna on nodejs, when I call:

```
let receipt = await methods.iotx.getReceiptByAction({actionHash: myHash})
```
I get following error:
```
Error: 13 INTERNAL: Failed to parse server response
ncalculator.js:71
code:13
details:"Failed to parse server response"
message:"13 INTERNAL: Failed to parse server response"
metadata:undefined
stack:"Error: 13 INTERNAL: Failed to parse server response\n    at Object.exports.createStatusError (/Users/simone/Source/web/iotexdelegate.com/node_modules/grpc/src/common.js:91:15)\n    at Object.onReceiveStatus (/Users/simone/Source/web/iotexdelegate.com/node_modules/grpc/src/client_interceptors.js:1204:28)\n    at InterceptingListener._callNext (/Users/simone/Source/web/iotexdelegate.com/node_modules/grpc/src/client_interceptors.js:568:42)\n    at InterceptingListener.onReceiveStatus (/Users/simone/Source/web/iotexdelegate.com/node_modules/grpc/src/client_interceptors.js:618:8)\n    at callback (/Users/simone/Source/web/iotexdelegate.com/node_modules/grpc/src/client_interceptors.js:845:24)"
```

This happens with `myHash`being an hash of type `grantReward` or `Transfer` did not try with other kind of actions

**Comment by puncsky** [comment]
thanks for the report! 

we may need to bump protos

**Comment by simonerom** [comment]
Any workaround / anything I can fix myself to make it work meanwhile?

**Comment by puncsky** [comment]
fixed in 0.10.0 https://github.com/iotexproject/iotex-antenna/commit/3c4de78f6a09830cd3a218756558135202ca3080
